kernel/kernel-api/src/main/java/io/delta/kernel/internal/snapshot/SnapshotManager.java

```diff
@@ -356,31 +359,32 @@ protected Optional<LogSegment> getLogSegmentForVersion(
         final List<FileStatus> checkpoints = checkpointsAndDeltas._1;
         final List<FileStatus> deltas = checkpointsAndDeltas._2;
 
-        logDebug(() ->
+        logger.atDebug().setMessage(() ->
```

Comment:
what's this `.setMessage` API? I googled around but couldn't find it. Could you link to it? Does it just take a in a supplier?